core/rio/api/src/main/java/org/eclipse/rdf4j/rio/RDFFormat.java

```diff
@@ -264,14 +264,22 @@
 				qValue -= 1;
 			}
 
-			for (String mimeType : format.getMIMETypes()) {
-				String acceptParam = mimeType;
+			if (RDFXML.equals(format)) {
+				// We explicitly dislike RDF/XML as it has limitations in what it can serialize. See #299.
+				qValue -= 4;
```

Comment:
It is possible with this change that RDF/XML gets excluded if exclusions are active below, as it doesn't support contexts, so -5, and it may not be the preferred format, so -2, which with -4 here, seems to add up to -11 which would make it go negative.

The goal of the issue wasn't to exclude RDF/XML, as it is far too popular for that, just to lower its priority.